### DISCUSSION TITLE
Popover component while Input is onfocus

### DISCUSSION THREAD

**Discussion by luiskabes-arch** [heroui-inc/heroui#3277] [Feedback]
Add a support to open the Popover component while Input component is onfocus

**Comment by wingkwong** [comment]
It's not clear based on this statement. Popover and input are two separate components. Do you mean autocomplete?

**Reply by luiskabes-arch** [reply]
Hello @wingkwong 

Thanks for the reply,
Yes, it's a separate component, what I want to achieve here is open the Popover component to display some information, but the Popover should only opened when the Input component is onfocus.

here is the preview
![Monosnap Search for: 2024-06-18 23-40-05](https://github.com/nextui-org/nextui/assets/85467724/b05969b1-e537-4816-b551-4058a23482e1)

**Reply by wingkwong** [reply]
Got it. What you can do is 

- for popover, use controlled one. See [here](https://nextui.org/docs/components/popover#controlled) for an example.
- in input, set the popover open in `onFocus`

**Reply by luiskabes-arch** [reply]
I'll give it a shot, btw is onFocus one of event from Input component?? because I can't see that on the documentation

**Reply by wingkwong** [reply]
`onFocus` is an event handler in React. We only document our NextUI-specific events in our documentation.
